source/install/install-kubernetes-mattermost.rst

```diff
@@ -5,7 +5,29 @@ Deploy a Mattermost Installation
 
 Deploying a Mattermost installation is very simple and just requires creating and applying a single manifest.
 
-**1. Create your installation manifest file**
+**1. (Enterprise Only) Create your Mattermost license secret**
```

Comment:
Could this be an independent point (not numbered) to clarify that it's only applicable to Enterprise customers?

Reply:
Not exactly sure how that would work but feel free to make a PR afterwards to update it :) I'm just going to merge this as is to get something out there